eosc/cmd/msigList.go

```diff
@@ -0,0 +1,62 @@
+// Copyright © 2018 EOS Canada <info@eoscanada.com>
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	eos "github.com/eoscanada/eos-go"
+	"github.com/spf13/cobra"
+)
+
+var msigListCmd = &cobra.Command{
+	Use:   "list [proposer]",
+	Short: "Shows the list of all active proposals for a given proposer in the eosio.msig contract.",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		api := getAPI()
+
+		proposer := toAccount(args[0], "proposer")
+
+		response, err := api.GetTableRows(
+			eos.GetTableRowsRequest{
+				Code:  "eosio.msig",
+				Scope: string(proposer),
+				Table: "approvals",
+				JSON:  true,
+			},
+		)
+		errorCheck("get table row", err)
```

Comment:
the problem with `GetTableRows` is that the list can overflow a single request, and then you need to query another time with `Offset` or `Limit`, etc.. 

This is good to start with, but we definitely need such a functionality even at the `eos-go` level.. so that people don't always have to reimplement that logic.

Reply:
That's good to get started.. I tihnk the limit is either 100 or 1000. but the thing is the ABI serializer deadline will kick in, and can return half of it because it times out!